README.md

```diff
@@ -28,10 +28,11 @@ The software requirements on the build machine are:
   * for seamless foreign arch emulation via qemu-user-static
 * `vagrant` with `virtualbox` (for non-linux targets)
 
-On Ubuntu 20.04, this can be installed via a single command:
-
-    sudo apt install -y python-yaml docker.io vagrant virtualbox p7zip-full
+These software requirements can be installed via a single command on Ubuntu:
 
+*  20.04: `sudo apt install -y python-yaml docker.io vagrant virtualbox p7zip-full`
+*  22.04: `sudo apt install -y python3-yaml docker.io vagrant virtualbox p7zip-full`
+*  
```

Comment:
```suggestion
On Ubuntu 22.04, this can be installed via a single command:

    sudo apt install -y python3-yaml docker.io vagrant virtualbox p7zip-full
```
I'd rather just document the current Ubuntu LTS, what do you think?

Reply:
Sorry, I somehow missed your response and  just seeing this now. I agree it makes sense to only document the latest release.